src/client/dfuse/il/intercept.h

```diff
@@ -118,8 +118,13 @@
 	} while (0);
 
 #else /* !IOIL_PRELOAD */
-#define IOIL_FORWARD_DECL(type, name, params)  \
-	extern type __real_##name params;
+/* clang-format off */
+#define IOIL_FORWARD_DECL(type, name, params)                                                      \
+	_Pragma("GCC diagnostic push")                                                             \
+	_Pragma("GCC diagnostic ignored \"-Wmissing-attributes\"")                                 \
```

Comment:
this fails on leap 15 clang, probably doesn't like the particular warning

Reply:
You might be able to fix it by also adding 

_Pragma("GCC diagnostic ignored \"-Wpragmas\"")

Reply:
another option here would be to fix the actual issue.  If an attribute is required (and consistent), we could just add another argument to IOIL_FORWARD_DECL